### PR TITLE
docs: Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by zyxiaoyuer**
ms-teams-vscode-extension	v4.0.3-rc.1

CLI BREAKING CHANGE:
  

Extension-toolkit BREAKING CHANGE:
 

CLI feat commits:
  

Extension-toolkit feat commits:

### CODE REVIEW

File: packages/vscode-extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 4.0.3 - Jul 25, 2022
+
+Incremental version for Teams Toolkit with multiple bugs fix and enhancements.
+
+Enhancement:
+
+- User can now switch Azure account or Azure subscription to provision could resource.
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/CHANGELOG.md:9]: typo, cloud resource

File: packages/vscode-extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 4.0.3 - Jul 25, 2022
+
+Incremental version for Teams Toolkit with multiple bugs fix and enhancements.
+
+Enhancement:
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/CHANGELOG.md:7]: We also have SPFx upgraded to v1.15.0 and would like to notify users about that. cc @xu-hong

File: packages/vscode-extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 4.0.3 - Jul 25, 2022
```
Comment (wenytang-ms) [inline packages/vscode-extension/CHANGELOG.md:3]: please adjust to tomorrow, Jul 25, 2022
thanks

File: packages/vscode-extension/CHANGELOG.md (hunk 1 above)
Comment (zyxiaoyuer) [reply packages/vscode-extension/CHANGELOG.md]: thanks. resolved.

File: packages/vscode-extension/CHANGELOG.md (hunk 2 above)
Comment (zyxiaoyuer) [reply packages/vscode-extension/CHANGELOG.md:7]: thank you, added.

File: packages/vscode-extension/CHANGELOG.md (hunk 3 above)
Comment (zyxiaoyuer) [reply packages/vscode-extension/CHANGELOG.md]: change to 26th, Thanks.

**Review by HuihuiWu-Microsoft — APPROVED** [review]
LGTM

**Review by wenytang-ms — APPROVED** [review]

**Review by tecton — APPROVED** [review]

**Review by yuqizhou77 — APPROVED** [review]

**Pull request merged**